### DISCUSSION TITLE
HTTP ERROR 500 - sdev 120

### DISCUSSION THREAD

**Discussion by sspirek** [community/community#102705] [GitHub Education]
### Hi everyone,

Hi everyone.. I'm in sdev120 learning python. we use github but i can maybe login, most often I can't. Most of my class has this issue. we keep getting errors. If we do multiple assignments at once. we get kicked out. how do we fix this? 
This page isn’t workinggithub-middleware.prod.eks.cengage.io is currently unable to handle this request.
HTTP ERROR 500

**Comment by HakimIno** [comment]
Response #2

**Comment by HVERNODU** [comment]
Hello,

I'm having a different problem in a separate class that is also using cengage to teach python where github won't even open thanks to an HTTP 500 error. This problem has been going on for over a week as of writing this. 
What has somewhat worked for me is using github during downtime, so I believe it to be a problem with the amount of traffic on github's servers.

**Comment by theferdelance** [comment]
Having the same problem from cengage

**Reply by cris-renee** [reply]
me too and no im a day late turning my work in

**Reply by localmemedealer** [reply]
same

**Reply by Sprintracer24** [reply]
same here. Although it was working 30 minutes ago

**Reply by Hawk19041** [reply]
This software barely works at all. Half the time I can't get on and the other half there is some weird bug or issue.

**Comment by AdrienneWilson** [comment]
I cannot access any of my html/css/internet programming class projects due to this same exact error

**Comment by georgeasnts** [comment]
Same here y'all, I am at a database class that also uses cengage as an online learning platform. Anyone has solved this issue?

**Comment by exwundee** [comment]
I'm in a Java class and I am also receiving this error.

**Comment by phuccoder2000** [comment]
Hi @sspirek. I think Alternative Access:
If there are alternative ways to access the resources you need (e.g., GitHub Classroom, GitHub Desktop, or direct Git commands), consider using those methods while the web interface issue is being resolved.
